src/app/config/index.ts

```diff
@@ -0,0 +1,2 @@
+export const NUMBER_OF_ITEMS_ON_DASHBOARD = 5
+export const NUMBER_OF_ITEMS_ON_SEPARATE_PAGE = 10
```

Comment:
Would be more descriptive with something like `TABLE_PAGE_SIZE`, but either way, we can change that later.